analyzers/src/SonarAnalyzer.CSharp/Facade/CSharpSyntaxFacade.cs

```diff
@@ -83,6 +83,11 @@ node switch
     public override ImmutableArray<SyntaxToken> FieldDeclarationIdentifiers(SyntaxNode node) =>
         Cast<FieldDeclarationSyntax>(node).Declaration.Variables.Select(x => x.Identifier).ToImmutableArray();
 
+    public override SyntaxKind[] ModifierKinds(SyntaxNode node) =>
+        node is TypeDeclarationSyntax typeDeclaration
+            ? typeDeclaration.Modifiers.Select(x => x.Kind()).ToArray()
+            : Array.Empty<SyntaxKind>();
```

Comment:
Please bump the coverage in `SyntaxFacadeTest` for C# and VB and the `else` branch. It should be easy with `SyntaxFactory` like in the other UTs there